docs/rules/require-prop-comment.md

```diff
@@ -0,0 +1,144 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/require-prop-comment
+description: require props to have a comment
+---
+# vue/require-prop-comment
+
+> require props to have a comment
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> ***This rule has not been released yet.*** </badge>
+
+## :book: Rule Details
+
+This rule enforces that every prop has a comment that documents it.
+
+<eslint-code-block :rules="{'vue/require-prop-comment': ['error']}">
+
+```vue
+<script>
+export default defineComponent({
+  props: {
+    // ✓ GOOD
```

Comment:
Please make sure that all examples from the docs are included in the tests.

Reply:
ok